README.md

```diff
@@ -1,45 +1,44 @@
 # Containerd Wasm Shims
-This project aims to provide containerd shim implementations that can run Wasm / WASI workloads using [runwasi](https://github.com/deislabs/runwasi) as a library. This means that by installing these shims onto Kubernetes nodes, we can add a runtime class to Kubernetes and schedule Wasm workloads on those nodes. Your Wasm pods and deployments can act just like container workloads!
+This project aims to provide containerd shim implementations that can run [Wasm](https://webassembly.org/) / [WASI](https://github.com/WebAssembly/WASI) workloads using [runwasi](https://github.com/deislabs/runwasi) as a library. This means that by installing these shims onto Kubernetes nodes, we can add a [runtime class](https://kubernetes.io/docs/concepts/containers/runtime-class/) to Kubernetes and schedule Wasm workloads on those nodes. Your Wasm pods and deployments can act just like container workloads!
 
-[runwasi](https://github.com/deislabs/runwasi) is a project that aims to run wasm workloads running on [Wasmtime](https://wasmtime.dev/), a fast and secure runtime for WebAssembly, which is managed by containerd.
+[runwasi](https://github.com/deislabs/runwasi) is a project that aims to run WASI workloads managed by [containerd](https://containerd.io/).
 
-## Quickstarts
+## Shims
 
-- [Start k3d and run a sample WASM application](./deployments/k3d/README.md#how-to-run-the-example).
-- [Create a Spin application on k3d](./containerd-shim-spin-v1/quickstart.md)
-- [Deploy a SpiderLightning application with k3d](./containerd-shim-slight-v1/quickstart.md)
-- [Deploy a Wasm Workers Server application with k3d](./containerd-shim-slight-v1/quickstart.md)
+This repo currently maintains four shims for Wasm application runtimes/frameworks:
 
-## Containerd Wasm Shims
-Each of the shims below leverage runwasi to provide the bridge between K8s and containerd.
+1. [Lunatic](https://github.com/lunatic-solutions/lunatic) - an Earlang-inspired runtime for fast, robust and scalable server-side Wasm application. 
+2. [Spin](https://github.com/fermyon/spin) - a developer tool for building and running serverless Wasm applications.
+3. [Slight](https://github.com/deislabs/spiderlightning) - a wasmtime-based runtime for running Wasm applications that use SpiderLightning (aks [WASI-Cloud-Core](https://github.com/WebAssembly/wasi-cloud-core)) capabilities
+4. [Wasm Workers Server](https://github.com/vmware-labs/wasm-workers-server) - a tool to develop and run serverless applications server on top of Wasm.
 
-### Spin shim
-The Spin shim, as the name implies, is powered by the [Fermyon Spin](https://github.com/fermyon/spin) engine. Spin is an open source framework for building and running fast, secure, and composable cloud microservices with WebAssembly.
 
-If you are curious, [here is the Spin shim source code](./containerd-shim-spin-v1).
+| **shim version** | v0.9 |
```

Comment:
this table should have a lead in that says "currently the most recent versions of are:" or something similar.

Reply:
Done. PTAL @squillace